app/assets/stylesheets/resource_curriculum_show.scss

```diff
@@ -1,42 +1,45 @@
 @import 'color';
 
-
-.resources_container a, .curriculums_container a {
-  color: $brick;
-  &:hover {
-    color: lighten( $brick, 10% );
-    text-decoration: underline;
+.resources_container, .curriculums_container {
+  a {
+    color: $brick;
+    &:hover {
+      color: lighten( $brick, 10% );
+      text-decoration: underline;
+    }
   }
-}
 
-li {
-  list-style-type: none;
-  font-size: 1rem;
-}
+  li {
+    list-style-type: none;
+    & a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Unnecessary parent selector (&)